app/js/components/cohorts/cohortModal.js

```diff
@@ -56,8 +57,8 @@ class ActionsComponent extends Component {
     handleSubmit(e) {
         e.preventDefault();
         const query = this.state.query;
-        if (query && !isNaN(this.state.queryId) && this.state.hasOwnProperty('name') &&
-            this.state.hasOwnProperty('description')) {
+        if (query && !isNaN(this.state.queryId) && e.target.name.value.length>0 &&
```

Comment:
Awesome PR @wanjikum and @paulupendo Please can you add spaces between `e.target.name.value.length` and `>` and `0`. This would make that snippet more readable.